packages/web/test/mocks/sample.teal

```diff
@@ -0,0 +1 @@
+int 1
```

Comment:
add empty line

Reply:
okay

Reply:
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/100185149/204323371-db0e35db-488d-4707-9fe3-bc8322183eb8.png">
this throws me error 
so I added it back

Reply:
an empty line causes the error?